main.nf

```diff
@@ -76,22 +76,22 @@ ch_synapse_files
 // Update Synapse URIs in input file with staged locations
 process update_input {
 
-  publishDir "${outdir}/${run_name}/",      mode: 'copy'
+  publishDir "${input_file.scheme}://${input_file.parent}/synstage/",  mode: 'copy'
+  publishDir "${outdir}/${run_name}/",          mode: 'copy'
 
   input:
-  path input    from input_file
-  val  exprs    from ch_synapse_sed
+  path "input.txt"    from input_file
```

Comment:
I like this change - I did not know that nextflow could rename input files.

Reply:
Yeah, it was more important for the output file to remain the same than the input file that's not published.